client/src/components/utils/SASSettings.ts

```diff
@@ -2,18 +2,25 @@
 // SPDX-License-Identifier: Apache-2.0
 import { workspace } from "vscode";
 
-export function isOutputHtmlEnabled(): boolean {
+function isOutputHtmlEnabled(): boolean {
   return !!workspace.getConfiguration("SAS").get("results.html.enabled");
 }
 
-export function getHtmlStyle(): string {
+function getHtmlStyle(): string {
   return workspace.getConfiguration("SAS").get("results.html.style");
 }
 
-export function isSideResultEnabled(): string {
+function isSideResultEnabled(): string {
   return workspace.getConfiguration("SAS").get("results.sideBySide");
 }
 
-export function isSinglePanelEnabled(): string {
+function isSinglePanelEnabled(): string {
   return workspace.getConfiguration("SAS").get("results.singlePanel");
 }
+
+export const SASSettings = {
```

Comment:
This is unnecessary. Importer can choose to write
```
import * as settings from "../utils/settings";
```
I don't think we need SAS prefix as all settings are for SAS extension.